### ISSUE TITLE
Label with transparent black background?

### ISSUE THREAD

**Issue by XenonCoder** [XenonCoder/TerraCraft#28]
It would be useful to have the game labels (fps and coords) of white color with a transparent black background to have better readability.

**Comment by benmoran56** [comment]
Perhaps it's a good time to make some simple classes for UI elements.

**Comment by XenonCoder** [comment]
@benmoran56 It would be nice to see a help screen to view the game controls. The user interface is important, once the basic functionality is defined, a new inventory system is added to increase the number of usable blocks and other improvements.

**Comment by benmoran56** [comment]
How do you imagine the help screen is accessed? By pressing a certain key in-game?

**Comment by XenonCoder** [comment]
Yes, the F1 key. I think it may be fine, Terracraft does not aspire to be a complex engine, so for other in-game features we can use the other keys later.

**Comment by benmoran56** [comment]
I created a sample help screen, in pull request #37. This example is using a new Scene to show some sample text. If you don't like this design, please feel free to reject it!

**Comment by XenonCoder** [comment]
@benmoran56 I like it, and as I said before we improve the graphic design later. The important part is the engine, adding images and sounds is the least complex thing. I'm testing everything, I take notes, and I make graphic contents to add. Thanks so much.

**Comment by Arne1303** [comment]
I think some background for that debug label would still be nice... I think I'll add that

**Comment by Arne1303** [comment]
I added that Background in PR #48!
